torch/utils/data/_utils/worker.py

```diff
@@ -161,6 +162,8 @@ def _worker_loop(dataset_kind, dataset, index_queue, data_queue, done_event,
         iteration_end = False
 
         watchdog = ManagerWatchdog()
+        p = psutil.Process()
```

Comment:
Should `psutil` be added to the list of required PyTorch dependencies? At the moment, it seems it's not required and various checks are made in the codebase to check if it's installed:
https://github.com/pytorch/pytorch/blob/21c38e17997171415a44c6ba578f621037d8ef30/test/test_dataloader.py